pkg/tsdb/elasticsearch/client/client_test.go

```diff
@@ -333,7 +334,8 @@ func TestClient(t *testing.T) {
 					So(sc.request, ShouldNotBeNil)
 					So(sc.request.Method, ShouldEqual, http.MethodPost)
 					So(sc.request.URL.Path, ShouldEqual, "/_msearch")
-					So(sc.request.URL.RawQuery, ShouldEqual, "max_concurrent_shard_requests=6")
+					So(sc.request.URL.RawQuery, ShouldContainSubstring, "max_concurrent_shard_requests=6")
+					So(sc.request.URL.RawQuery, ShouldContainSubstring, "my_param=my_value")
```

Comment:
A test including a parameter that requires URL encoding would be good to verify that it's properly URL encoded. Right now I think there's no handling of URL encoding.